doc/classes/ProjectSettings.xml

```diff
@@ -2554,6 +2554,7 @@
 			[b]Note:[/b] This setting is only effective when using the Compatibility rendering method, not Forward+ and Mobile.
 		</member>
 		<member name="rendering/limits/spatial_indexer/threaded_cull_minimum_instances" type="int" setter="" getter="" default="1000">
+			The minimum number of instances for which to run the culling computations on multiple threads. Below this value, culling is done on a single thread.
```

Comment:
This is accurate, but I think the language can be improved to be a little bit clearer
```suggestion
			The minimum number of instances that must be present in a scene to enable culling computations on multiple threads. If a scene has fewer instances than this number, culling is done on a single thread.
```

Reply:
Oh yeah, that sounds a lot clearer